wordpress-mobile/WordPress-Android
# Use view binding in stats

This PR introduces ViewBindings in the stats, stats detail and stats view all screens. There were several tricky things to figure out. 
- When using the FragmentContainerView we now can't initialize the activity action bar in activity `onCreate` so I've moved it to fragment `onViewCreated`. 
- The `@SuppressLint("ClickableViewAccessibility")` is there because the way we catch the touches on the view pager makes the AS think that it affects accessibility. As far as I could see this is not the case so I've added the suppression. Let me know if you think otherwise. 
- I'm not sure my approach to `binding` is the best. I'm trying to pass it around as a parameter as much as possible and only use the field when the binding needs to be accessible from other methods than `onViewCreated`. When using `views` this is the safest approach because you can be sure the view is there. Here I'm not 100% sure.

To test:
- Go to Stats and notice it's loaded correctly
- Notice the action bar has the correct title and the back button is active
- Click on `View more` on a block on a site with a lot of data (you might need to go to the `Year` view)
- Notice the "View all" screen is loaded correctly
- Notice the action bar has the correct title and the back button is active
- Go to My Site/Blog posts
- Click on the overflow menu on one of the older blog posts (not an X-post) and go to "Post Stats"
- Notice the screen is loaded correctly
- Notice the action bar has the correct title and the back button is active

PR submission checklist:
